CHANGELOG.md

```diff
@@ -5,3 +5,4 @@
 * Add new app management commands: `apps:create`, `apps:list`, `apps:sdkconfig`.
 * Improve `init` command to be able to create a new project.
 * Automatically choose a port for the Firestore emulator to serve WebChannel traffic.
+* Deprecate `list` command.
```

Comment:
Maybe

```
* Mark `list` command as deprecated.
```

Since it's not being _removed_, I don't want to super scare anybody. Marking it as deprecated wouldn't be a breaking change - removing it would be.

Reply:
(eh, on second thought, maybe this is fine. The "mark" wording is a little less aggressive though... let me know your thoughts)

Reply:
I think your suggestion makes a lot of sense. Updating